tweb/src/main/java/io/fundreqest/platform/tweb/infrastructure/mav/Iniitalizer.java

```diff
@@ -0,0 +1,52 @@
+package io.fundreqest.platform.tweb.infrastructure.mav;
+
+import io.fundreqest.platform.tweb.fund.FundController;
+import io.fundreqest.platform.tweb.request.RequestController;
+import io.fundrequest.core.request.RequestService;
+import io.fundrequest.core.request.statistics.StatisticsService;
+import io.fundrequest.platform.faq.FAQService;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Iniitalizer {
+
+    @EventListener
+    public void onApplicationEvent(ContextRefreshedEvent event) {
+        initialize(event.getApplicationContext());
+    }
+
+    private void initialize(ApplicationContext context) {
+        initializeRequests(context);
+        initializeTechnologies(context);
+        initializeProjects(context);
+        initializeStatistics(context);
+        initializeFaqs(context);
+    }
+
+    private void initializeFaqs(ApplicationContext context) {
```

Comment:
The FAQ already get initialised by the `@Scheduled(fixedRate = DAY_IN_MILLIS)` on `GithubFAQServiceImpl.refreshFAQs()`. This will run once at startup and daily after that.